src/open_company_web/api.cljs

```diff
@@ -84,3 +84,22 @@
           (fn [response]
             (let [body (if (:success response) (json->cljs (:body response)) {})]
               (flux/dispatch dispatcher/report body))))))))
+
+(defn save-or-create-company [symbol data]
+  (when (and symbol data)
+    (let [data (dissoc data :headcount)
+          data (dissoc data :finances)
+          data (dissoc data :compensation)
+          data (dissoc data :links)
```

Comment:
also best to to treat let's as if there is single assignment (like Erlang):

```clojure
(let [profile-data (apply dissoc data [:headcount :finances :compensation :links])]
```

rather than:

```clojure
(let [data (apply dissoc data [:headcount :finances :compensation :links])]
```

ie. don't reuse the `data` variable.